runtime/features/org.jboss.tools.runtime.core.feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <feature
       id="org.jboss.tools.runtime.core.feature"
       label="%featureName"
-      version="3.5.100.qualifier"
+      version="3.5.101.qualifier"

```

Comment:
Upversioning is generally done with 100 increment to keep space for urgent security fix releases